run-all-tests

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I wonder if this script should go in `./bin`.

Reply:
That seems like a clean way to do it, but I don't have any strong opinions either way.